access.md

```diff
@@ -15,32 +15,15 @@ The serial console of a virtual machine can be accessed by using the `console`
 command:
 
 ```bash
-$ virtctl console -s http://mycluster:8184 testvm
+$ virtctl console --kubeconfig=$KUBECONFIG testvm
 ```
 
-### Accessing the graphical console
+### Accessing the graphical console (VNC)
 
 Accessing the graphical console of a virtual machine is usually done through
-SPICE, which requires a SPICE client like `remote-viewer`. Once the tool is
+VNC, which requires a VNC client like `remote-viewer`. Once the tool is
 installed you can access the graphical console using:
 
 ```bash
-$ virtctl spice -s http://mycluster:8184 testvm
+$ virtctl vnc --kubeconfig=$KUBECONFIG testvm
 ```
-
-Optional `--proxy` (e.g. `http://192.168.200.2:1234`) can be used. This is
-useful when you are outside of the cluster or a non-reachable proxy is provided
-by the system.
-
-#### Grapical console connection details
-
-Instead of establishing the connection, `virtctl` can provide the connection
-details, which can then be used with a tool of your choice to manually
-establish the connection. The connection details can be retrieved by running:
```

Comment:
This isn't related to your changes, but it got me thinking.

We could allow an option to run virtctl as a local VNC server in the future which would allow people to use whatever VNC client they want.

Reply:
Nifty idea :)

We could take the same approach for spice i.e. offering a local spice proxy (and vnc proxy).